src/components/users/UserRegForm.js

```diff
@@ -0,0 +1,204 @@
+import React, { Component } from 'react'
+import Axios from 'axios'
+import { debounce } from 'throttle-debounce'
+import emailValidator from 'email-validator'
+// material
+import TextField from 'material-ui/TextField'
+import RaisedButton from 'material-ui/RaisedButton'
+import Snackbar from 'material-ui/Snackbar'
+// custom
+import { URL_USERS } from '../../constants/Constants'
+
+class UserRegForm extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      registerButtonDisabled: true,
+      snackAutoHideDuration: 2000,
+      snackMessage: 'User registered',
+      snackOpen: false
+    }
+  }
+
+  validate = () => {
+    this.setState({
+      registerButtonDisabled: !(this.emailIsValid() &
+                this.passIsValid() &
```

Comment:
Putting an operator in a front of the line is a little bit better for readability.